src/pages/index.tsx

```diff
@@ -114,6 +124,8 @@ const Index: FC = () => {
   }, [isLoading, response]);
 
   useEffect(() => {
+    console.log("in use effect: ", checkThemeHistory(document.cookie, query))
```

Comment:
Please avoid adding console statements to production

Reply:
No problem I'll go ahead and remove that